fix: Update instructions following the Airbyte Cloud integration (#57)
KYVENetwork/docs: docs/data_engineers/accessing_data/elt_pipeline/overview.md

format, data analysts and engineers can now confidently source KYVE data without worrying about its validity or
reliability.

The sections that follow were created to help you understand how to setup the KYVE source connector of Airbyte in order to retrieve data. There are two options that you can test the ELT connector of KYVE.

## Airbyte Local Deployment

For users that wants to experiment locally with the KYVE data it is recommended the Local Deployment approach of Airbyte. This option requires some configuration steps in order to run locally the Airbyte app. Detailed instructions are presented in [Airbyte Local Deployment](airbyte_local_deployment) section.

## Airbyte Cloud

> You can connect to the [Airbyte Cloud](https://airbyte.com/airbyte-cloud) platform and try the [KYVE Source Connector](https://cloud.airbyte.com/workspaces/2a2bfca3-c289-48af-a11d-1a9ad69ea187/source/new-source/60a1efcc-c31c-4c63-b508-5b48b6a9f4a6).

Reviewer: Is this workspace publicly available? The link doesn't work for me.

Author: You were right this link is not working for every user, so we will not use a direct link.

> This option does not require any local setup. For setting up destination you could follow the documentation inside the Airbyte cloud page. [Snowflake](elt_destinations/snowflake) section presents a destination example that could be used with cloud.

Reviewer: ```suggestion
This option does not require any local setup. For setting up a destination follow the documentation in the Airbyte cloud page. For example, the [Snowflake](elt_destinations/snowflake) section presents one such destination that could be used with cloud.
```
